cmd/nerdctl/login.go

```diff
@@ -123,9 +123,6 @@ func loginAction(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("error saving credentials: %w", err)
 	}
 
-	if _, err = loginClientSide(ctx, cmd, *authConfig); err != nil {
-		return err
-	}
```

Comment:
The second call has a different `*authConfig`, so I'm not sure we can safely remove this.

Reply:
this second login can be based on : 

1.  empty / Password (Identity token) --> useless because we previously did a login with username/password
2. Username / Password  (not Identity token) --> this is done previously 

I wondering what is the reason behind making double login operation especially when filled the `CredentialsStore` with creds